### DISCUSSION TITLE
Downloading history data from Nightscout to xDrip

### DISCUSSION THREAD

**Discussion by Pipperoo169** [NightscoutFoundation/xDrip#2706] [General]
My xdrip has eventually been uninstalled. When I installed it again, I didn't find any history monitoring data there. The data starts from the moment of new setting up was done. Is there any way to bring all history data from Nightscout back to xdrip now?

Thanks,
Natalia

**Comment by Navid200** [comment]
This happens a lot when someone uninstalls xDrip and is surprised to find out that the history is gone.
This is why I have everywhere on my site (https://navid200.github.io/xDrip/) warnings against uninstalling xDrip.

But, the best solution, in my opinion, is to maintain a single copy of the database updated daily by default in the phone internal storage.

My understanding is that you can enable an xDrip setting to save the database daily.  But, that will keep adding files everyday without deleting the previous ones.

At least, it's great that you have your data stored on Nightscout.
I don't know how to import data from Nightscout to xDrip.

**Comment by Pipperoo169** [comment]
Thank you for clarification. Luckily the history data is also stored on the phones of followers.

**Reply by Navid200** [reply]
Please explain what you mean by that.
Are you saying if you install xDrip on a phone and set it up to be a Nightscout follower, all the data from Nightscout is automatically transferred to xDrip?

**Comment by Pipperoo169** [comment]
No exactly. I mean my xDrip follower had been installed together with xDrip master and when xDrip master was uninstalled, xDrip follower still kept history data which is a way out.
